src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class StringValidatorTests
+    {
+        [Fact]
+        public void Constructor_MinLength()
+        {
+            //We don't expect this to fail
```

Comment:
@hughbe For this class, I don't think there is a way to verify if the constructor worker with using an assert.  The best way I can think of is to test the _minLength readonly member.  But that is protected and the assert can't reach it.

Another way I could test would to see if the validate method threw an exception if I passed it in a string that was too small, but that won't have an assert either.

Should I remove the tests for the constructor?